common/lib/calc/calc/tests/test_calc.py

```diff
@@ -482,17 +471,16 @@ def test_variable_case_sensitivity(self):
             8.0
         )
 
-        variables = {'t': 1.0}
-        self.assertEqual(calc.evaluator(variables, {}, "t"), 1.0)
-        self.assertEqual(calc.evaluator(variables, {}, "T"), 1.0)
+        variables = {'E': 1.0}
+        self.assertEqual(calc.evaluator(variables, {}, "E"), 1.0)
```

Comment:
This line has to go. The code gives unpredictable results when there is a name collision due to case insensitivity. This is an old issue, and not one that anybody actually runs into. Also, we should not be testing for unpredictable results.